src/Resources/config/core.php

```diff
@@ -166,6 +166,8 @@
             ->tag('container.service_subscriber')
             ->call('setContainer', [new ReferenceConfigurator(ContainerInterface::class)])
 
+        ->alias(CRUDController::class, 'sonata.admin.controller.crud')
```

Comment:
@VincentLanglet can you try making this alias public?

Reply:
It works ! thanks